build.gradle

```diff
@@ -203,7 +204,9 @@ dependencies {
 
   functionalTestImplementation sourceSets.main.runtimeClasspath
 
+  smokeTestImplementation group: 'io.rest-assured', name: 'rest-assured', version: versions.restAssured
```

Comment:
hmm, probably shouldn't add dependencies that are not used. you could add a ”real” test that hits the root endpoint with a TODO around it saying implement a real test?

Reply:
worked without this so removed.